.github/workflows/kernel.yml

```diff
@@ -68,13 +68,6 @@ jobs:
           echo "TARGET=$TARGET" >> "$GITHUB_ENV"
           echo "SUBTARGET=$SUBTARGET" >> "$GITHUB_ENV"
 
-      - name: Update & Install feeds
```

Comment:
There are Kernel modules in the feeds which are no longer tested once this is removed. I'm fine removing this but then again we should check if changed Kernels break other peoples packages.

Another option could be to use GitHubs mirrors of our feeds instead of git.openwrt.org

Reply:
I think we are only building the kernel modules which are activated by default and not all of them . None of the kernel modules from the feed should be active by default. 
You have to add this to the configuration to build all kernel modules: `CONFIG_ALL_KMODS=y`

Reply:
We set the configuration option on our buildbots

Reply:
No it does not use the configuration the build bots are using.
It uses this configuration for example:
````
CONFIG_TARGET_apm821xx=y
CONFIG_TARGET_apm821xx_nand=y
CONFIG_TARGET_apm821xx_nand_Default=y
CONFIG_DEVEL=y
CONFIG_BUILD_NLS=y
CONFIG_EXTERNAL_TOOLCHAIN=y
CONFIG_EXTERNAL_TOOLCHAIN_LIBC_USE_MUSL=y
CONFIG_LIBC_FILE_SPEC="./powerpc-openwrt-linux-musl/lib/ld-musl-powerpc.so.1 ./powerpc-openwrt-linux-musl/lib/libc.so"
CONFIG_LIBC_ROOT_DIR="/__w/openwrt/openwrt/openwrt/openwrt-toolchain-apm821xx-nand_gcc-11.3.0_musl.Linux-x86_64/toolchain-powerpc_464fp_gcc-11.3.0_musl"
CONFIG_LIBGCC_FILE_SPEC="./powerpc-openwrt-linux-musl/lib/libgcc_s.so ./powerpc-openwrt-linux-musl/lib/libgcc_s.so.1"
CONFIG_LIBGCC_ROOT_DIR="/__w/openwrt/openwrt/openwrt/openwrt-toolchain-apm821xx-nand_gcc-11.3.0_musl.Linux-x86_64/toolchain-powerpc_464fp_gcc-11.3.0_musl"
CONFIG_LIBPTHREAD_FILE_SPEC="./lib/libpthread{-*.so,.so.*}"
CONFIG_LIBPTHREAD_ROOT_DIR="/__w/openwrt/openwrt/openwrt/openwrt-toolchain-apm821xx-nand_gcc-11.3.0_musl.Linux-x86_64/toolchain-powerpc_464fp_gcc-11.3.0_musl"
CONFIG_LIBRT_FILE_SPEC="./lib/librt{-*.so,.so.*}"
CONFIG_LIBRT_ROOT_DIR="/__w/openwrt/openwrt/openwrt/openwrt-toolchain-apm821xx-nand_gcc-11.3.0_musl.Linux-x86_64/toolchain-powerpc_464fp_gcc-11.3.0_musl"
CONFIG_LIBSTDCPP_FILE_SPEC="./powerpc-openwrt-linux-musl/lib/libstdc++.so ./powerpc-openwrt-linux-musl/lib/libstdc++.so.6 ./powerpc-openwrt-linux-musl/lib/libstdc++.so.6.0.29 ./powerpc-openwrt-linux-musl/lib/libstdc++.so.6.0.29-gdb.py"
CONFIG_LIBSTDCPP_ROOT_DIR="/__w/openwrt/openwrt/openwrt/openwrt-toolchain-apm821xx-nand_gcc-11.3.0_musl.Linux-x86_64/toolchain-powerpc_464fp_gcc-11.3.0_musl"
CONFIG_PACKAGE_libstdcpp=y
CONFIG_STRIP_ARGS="--strip-all"
CONFIG_TARGET_NAME="powerpc-openwrt-linux-musl"
CONFIG_TOOLCHAIN_BIN_PATH="./usr/bin ./bin"
CONFIG_TOOLCHAIN_INC_PATH="./usr/include ./include"
CONFIG_TOOLCHAIN_LIBC="musl"
CONFIG_TOOLCHAIN_LIB_PATH="./usr/lib ./lib"
CONFIG_TOOLCHAIN_PREFIX="powerpc-openwrt-linux-musl-"
CONFIG_TOOLCHAIN_ROOT="/__w/openwrt/openwrt/openwrt/openwrt-toolchain-apm821xx-nand_gcc-11.3.0_musl.Linux-x86_64/toolchain-powerpc_464fp_gcc-11.3.0_musl"
CONFIG_USE_EXTERNAL_LIBC=y
# CONFIG_USE_SSTRIP is not set
CONFIG_USE_STRIP=y
````

The build bots are using this configuration: https://downloads.openwrt.org/snapshots/targets/apm821xx/nand/config.buildinfo

Reply:
Sorry my point was that the build bots are using the all kmods option and so should the CI. I think using GitHub repo mirrors fixes the issue in a better way than ignoring those external kmods

Reply:
I will try to remove the feed URLs using sed in the feeds.conf instead of deleting it.
I think we do not use any package from the feeds in these builds.